installer/templates/phx_web/templates/page/index.html.eex

```diff
@@ -3,6 +3,49 @@
   <p>A productive web framework that<br/>does not compromise speed and maintainability.</p>
 </section>
 
+<section class="row">
+  <article class="column">
+    <h3>New in v1.4</h3>
+    <ul>
+      <li>
+        <a href="https://github.com/phoenixframework/phoenix/pull/2734">
+          Jason is now the default JSON library instead of Poison
+        </a>
+      </li>
+      <li>
+        <a href="https://github.com/phoenixframework/phoenix/pull/2734">
+          Router helpers now use <code>Routes.my_path</code> instead of <code>my_path</code>
+        </a>
+      </li>
+      <li>
+        <a href="https://github.com/phoenixframework/phoenix/pull/2766">
+          New JavaScript presence API
+        </a>
+      </li>
+      <li>
+        <a href="https://github.com/phoenixframework/phoenix/pull/2621">
+          Cowboy 2 Support (Change cowboy version to 2.4 in mix.exs to activate)
+        </a>
+        <ul>
+          <li>
+            <a href="https://github.com/phoenixframework/phoenix/pull/2801">
+              <code>Router.Helpers.static_push/2</code> function for HTTP/2 server push
+          </li>
+          </li>
+        </ul>
+      </li>
+      <li>
+        <a href="https://github.com/phoenixframework/phoenix/pull/2779">
+          Webpack 4 has been replaced by brunch
```

Comment:
Other way round right? Brunch has been replaced with Webpack 4

Reply:
Definitely the other way round! Thanks.